optimade/server/tests/test_more_structures.json

```diff
@@ -1,5 +1,5 @@
 [{
-  "task_id": "db_1234567",
+  "task_id": "db/1234567",
```

Comment:
Does this work with the filter language as it is?

Reply:
(for e.g. `structures/db/1234567`)

Reply:
It does not. But I guess that is not the point of this PR - but rather to make everything up to date with 0.10.0. Then with the merging of #66 this should work :)

Reply:
:+1: I changed the test data as `db/1234-567` also fails with the current version.